File: Backend/C#/SuperHero.Domain/Model/Powerstats.cs
using SuperHero.Domain.Model.BaseModel;
using System.Text.Json.Serialization;

namespace SuperHero.Domain.Model
{
    public class Powerstats 
    {

        [JsonPropertyName("intelligence")]
        public string Intelligence { get; set; }

        [JsonPropertyName("strength")]
        public string Strength { get; set; }

        [JsonPropertyName("speed")]
        public string Speed { get; set; }

        [JsonPropertyName("durability")]
        public string Durability { get; set; }

        [JsonPropertyName("power")]
        public string Power { get; set; }

        [JsonPropertyName("combat")]
        public string Combat { get; set; }
    }
}

File: Backend/JAVA/application/src/main/java/com/superhero/domain/model/Powerstats.java
package com.superhero.domain.model;

import java.io.Serializable;

import com.fasterxml.jackson.annotation.JsonProperty;

public final class Powerstats implements Serializable{

    @JsonProperty("intelligence")
    public String intelligence;

    @JsonProperty("strength")
    public String strength;

    @JsonProperty("speed")
    public String speed;

    @JsonProperty("durability")
    public String durability;

    @JsonProperty("power")
    public String power;

    @JsonProperty("combat")
    public String combat;
    

    public Powerstats() {
    }

    public String getIntelligence() {
        return intelligence;
    }

    public void setIntelligence(String intelligence) {
        this.intelligence = intelligence;
    }

    public String getStrength() {
        return strength;
    }

    public void setStrength(String strength) {
        this.strength = strength;
    }

    public String getSpeed() {
        return speed;
    }

    public void setSpeed(String speed) {
        this.speed = speed;
    }

    public String getDurability() {
        return durability;
    }

    public void setDurability(String durability) {
        this.durability = durability;
    }

    public String getPower() {
        return power;
    }

    public void setPower(String power) {
        this.power = power;
    }

    public String getCombat() {
        return combat;
    }

    public void setCombat(String combat) {
        this.combat = combat;
    }

}
